main/src/main/java/com/sedmelluq/discord/lavaplayer/source/youtube/DefaultYoutubeTrackDetailsLoader.java

```diff
@@ -30,13 +26,8 @@
 
   private static final String AGE_VERIFY_REQUEST_URL = "https://www.youtube.com/youtubei/v1/verify_age?key=AIzaSyAO_FJ2SlqU8Q4STEHLGCilw_Y9_11qcW8";
   private static final String AGE_VERIFY_REQUEST_PAYLOAD = "{\"context\":{\"client\":{\"clientName\":\"WEB\",\"clientVersion\":\"2.20210302.07.01\"}},\"nextEndpoint\":{\"urlEndpoint\":{\"url\":\"%s\"}},\"setControvercy\":true}";
-
-  private static final String[] EMBED_CONFIG_PREFIXES = new String[]{
-      "'WEB_PLAYER_CONTEXT_CONFIGS':",
-      "WEB_PLAYER_CONTEXT_CONFIGS\":",
-      "'PLAYER_CONFIG':",
-      "\"PLAYER_CONFIG\":"
-  };
+  private static final String PLAYER_REQUEST_URL = "https://www.youtube.com/youtubei/v1/player?key=AIzaSyAO_FJ2SlqU8Q4STEHLGCilw_Y9_11qcW8";
```

Comment:
I would store this key in a constant so that it can be reused for all requests

Reply:
Yes, I said about this in comment